addons/html_editor/static/src/main/list/list_plugin.js

```diff
@@ -141,7 +142,17 @@ export class ListPlugin extends Plugin {
     onInput() {
         const selection = this.shared.getEditableSelection();
         const blockEl = closestBlock(selection.anchorNode);
-        const stringToConvert = blockEl.textContent.substring(0, selection.anchorOffset);
+        const leftDOMPath = leftLeafOnlyNotBlockPath(selection.anchorNode);
```

Comment:
You should return early from the `onInput` if `ev.data !== " "` just like you did in previous commit because I don't think we should call `leftLeafOnlyNotBlockPath` for every input event, as it can be a costly operation. What do you think?

Reply:
Done :+1: